### ISSUE TITLE
Add polylang string translation support

### ISSUE THREAD

**Issue by sybrew** [sybrew/the-seo-framework#257]
From [Jordy's comments](https://wordpress.org/support/topic/the-seo-framework-polylang-meta-title-description/):

Polylang doesn't support string translations for TSF's input fields (like titles and descriptions) out of the box.

[In my comment](https://wordpress.org/support/topic/the-seo-framework-polylang-meta-title-description/#post-9885298) I've tested out a resolution which could be included in TSF.
```php
add_filter( 'the_seo_framework_custom_field_description', 'my_tsf_pll__' );
add_filter( 'the_seo_framework_generated_description', 'my_tsf_pll__' );
add_filter( 'the_seo_framework_pro_add_title', 'my_tsf_pll__' ); // this will change in 3.1 due to its inefficiency.
function my_tsf_pll__( $input ) {
	return $input && function_exists( 'pll__' ) ? pll__( $input ) : $input;
};
```

However, titles don't always work as intended. This is because we can't filter title parts.
So, for that, I first want #202 to be completed.

**Comment by MikeiLL** [comment]
Thanks for your awesome plugin!

When TSF is deactivated the Meta HTML Title is `l10n`ed, but when plugin is enabled Title is in Default language.

It looks like you have [added these filters](https://github.com/sybrew/the-seo-framework/blob/18416c0c90bc50eeb1bbbbbb8d727590d64c23a9/inc/compat/plugin-polylang.php) in version `3.1.0`. I'm using version `3.2.1`

I had initially tried manually adding some of the filters:

```
the_seo_framework_custom_field_description
the_seo_framework_title_from_custom_field
the_seo_framework_generated_description
the_seo_framework_pro_add_title
the_seo_framework_pre_get_document_title
the_seo_framework_wp_title
the_seo_framework_title_from_custom_field
the_seo_framework_title_from_generation
```

None of them return a translated string when I return `pll__( $input )`.

Then I saw that the functionality was now built into the plugin. The description of the site as well as page for Meta as well as og tags are translating as expected, however, the `Page Title` and `og:title` tags are only displaying the default language strings.

I have tried with Page SEO Settings Meta Title populated as well as empty, and the TSF settings are populated.

Is this the expected behavior? Sorry for the SPAM, but am I missing something?
